### ISSUE TITLE
Bug in parsing json objects as query params

### ISSUE THREAD

**Issue by DenuxPlays** [swagger-api/swagger-ui#9755]
### Q&A (please complete the following information)
 - OS: Fedora 39
 - Browser: Chromium (Brave)
 - Version: Brave: v1.64.109
 - Method of installation: Via [rust-crate](https://github.com/juhaku/utoipa/tree/master/utoipa-swagger-ui)
 - Swagger-UI version: 5.11.0
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
I suspect that this is an excaping issue.
As when you try to send the example json (see json-spec below) my API endpoint gets:
```json
should: [object Object]
```
Doing the same request via tools like Insomnia my API gets the correct json.
I read about it that some js magic can produce the `[object Object]`.

Example Swagger/OpenAPI definition:
```json
 "/api/v1/transaction/search?page={page}&size={size}&search_query={search_query}": {
      "get": {
        "tags": [
          "Transaction"
        ],
        "operationId": "search",
        "parameters": [
          {
            "name": "page",
            "in": "path",
            "description": "The page number. Default: 1",
            "required": true,
            "schema": {
              "type": "integer",
              "format": "int64"
            }
          },
          {
            "name": "limit",
            "in": "path",
            "description": "The number of items per page. Default: 50",
            "required": true,
            "schema": {
              "type": "integer",
              "format": "int64"
            }
          },
          {
            "name": "search_query",
            "in": "path",
            "description": "SearchQuery object as a query string.!!!WARNING:!!! Swagger-ui has a bug you cannot test it via swagger!!!WARNING!!!",
            "required": true,
            "schema": {
              "type": "object",
              "properties": {
                "should": {
                  "type": "object",
                  "additionalProperties": {

                  },
                  "nullable": true
                },
                "should_not": {
                  "type": "object",
                  "additionalProperties": {

                  },
                  "nullable": true
                },
                "sort_by": {
                  "type": "object",
                  "additionalProperties": {
                    "$ref": "#/components/schemas/SortOrder"
                  },
                  "nullable": true
                }
              },
              "example": "{\n  \"should\": {\n    \"name\": \"Test\"\n  },\n  \"should_not\": {\n    \"FieldB\": \"value that it should not have\"\n  },\n  \"sort_by\":{\n    \"name\": \"Asc\"\n  }\n}"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "Successfully searched Transactions.",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/PaginatedTransaction"
                }
              }
            }
          },
          "400": {
            "description": "Provided data isn't valid.",
            "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "required": [
                    "api_code",
                    "details"
                  ],
                  "properties": {
                    "api_code": {
                      "$ref": "#/components/schemas/ApiCode"
                    },
                    "details": {
                      "type": "string"
                    },
                    "reference": {
                      "allOf": [
                        {
                          "$ref": "#/components/schemas/SerializableStruct"
                        }
                      ],
                      "nullable": true
                    }
                  }
                }
              }
            }
          },
          "401": {
            "description": "Unauthorized. Can be caused by: invalid credentials, expired token, missing token, malformed Authorization header, etc.",
            "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "required": [
                    "api_code",
                    "details"
                  ],
                  "properties": {
                    "api_code": {
                      "$ref": "#/components/schemas/ApiCode"
                    },
                    "details": {
                      "type": "string"
                    },
                    "reference": {
                      "allOf": [
                        {
                          "$ref": "#/components/schemas/SerializableStruct"
                        }
                      ],
                      "nullable": true
                    }
                  }
                }
              }
            }
          },
          "500": {
            "description": "Internal server error. Like DbError, HashingError, etc.",
            "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "required": [
                    "api_code",
                    "details"
                  ],
                  "properties": {
                    "api_code": {
                      "$ref": "#/components/schemas/ApiCode"
                    },
                    "details": {
                      "type": "string"
                    },
                    "reference": {
                      "allOf": [
                        {
                          "$ref": "#/components/schemas/SerializableStruct"
                        }
                      ],
                      "nullable": true
                    }
                  }
                }
              }
            }
          }
        },
        "security": [
          {
            "bearer_token": []
          }
        ]
      }
    }
```

Swagger-UI configuration options:
///

### To reproduce...

1. Add the spec-snippet from the endpoint I provided
2. Create an api that listens on the endpoint
3. set up the endpoint that it prints the query params
4. execute the "try it" feature via swagger-ui
5. see the scuffed query string.
6. (optional) to the same with insomnia

### Expected behavior
Send the url-encoded json.

### Screenshots
Swagger:
![image](https://github.com/swagger-api/swagger-ui/assets/83671398/002a8c7d-c7e0-4e5b-9f19-0e003be721cb)

Insomnia:
![image](https://github.com/swagger-api/swagger-ui/assets/83671398/39aa38e6-73ad-442c-a8c9-79204e42e508)

### Additional context or thoughts
///

**Comment by glowcloud** [comment]
Addressed in https://github.com/swagger-api/swagger-js/pull/3446

**Comment by DenuxPlays** [comment]
> Addressed in [swagger-api/swagger-js#3446](https://github.com/swagger-api/swagger-js/pull/3446)

Nice thank you.
Will this be patched in the next ui release?
